### ISSUE TITLE
Python: Support for Specifying AI Services in Prompt Configuration Files 

### ISSUE THREAD

**Issue by anqi-zhao** [microsoft/semantic-kernel#4450]
From: https://github.com/microsoft/semantic-kernel/pull/4434

When I have multiple AI services, I want to be able to specify the AI service used by the semantic plugin.

For example:
```python
kernel.add_text_completion_service(
    'text_completion_1',
    OpenAITextCompletion(
        ai_model_id = text_completion_model_id,
        async_client= async_client
    )
)

kernel.add_text_completion_service(
    'text_completion_2',
    OpenAITextCompletion(
        ai_model_id = text_completion_model_id,
        async_client= async_client
    )
)
```
In this case, I can specify the name of the AI service in the config.json of the semantic plugin.

```json
{
    ...
    "default_services": ["text_completion_2"]
    ...
}
```

**Comment by matthewbolanos** [comment]
We plan to align with the .NET implementation of the SDK where you can define multiple models as execution settings for a prompt.

**Comment by HuskyDanny** [comment]
Any plan on this? I think it is a key feature when it comes to production use where model availability, network, region, cost are taken into account. 

And right now it seems you could specify service_id in AzureChatRequestSettings and default_service in PromptTemplateConfig. But there is no switch under the hood?

**Comment by eavanvalkenburg** [comment]
This will be partly solved by #4565 in combination with #4631

**Comment by eavanvalkenburg** [comment]
This should work with the latest code, will be in v1beta

**Comment by moonbox3** [comment]
This is included as of #5077, closing work item.
